app/views/cloud_tenant/edit.html.haml

```diff
@@ -1,4 +1,8 @@
-%form#form_div{:name => "angularForm", 'ng-controller' => "cloudTenantFormController"}
+%form#form_div{:name => "angularForm",
+                'ng-controller' => "cloudTenantFormController as vm",
```

Comment:
Fixed alignment. I use tabs instead of spaces, so it wasn't apparent at first. Hope it's okay now.